redhat-appstudio/service-provider-integration-operator
# Add common service monitor for prometheus

### What does this PR do?
Adds service monitor and repates stuff for Prometheus

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-533

### How to test this PR?

 - Minikube:
        Install SPI on minikube. Run `./hack/deploy-minikube-monitoring.sh`. Verify after 5 min there is a data on Grafana dashboards.

 - Openshift
       Add resource ` - https://github.com/mshaposhnik/service-provider-integration-operator/config/monitoring/prometeus/base?ref=b6ee527ac21c6ce5f736548d8d88eaee6c3f96b6` onto `/infra-deployments/components/spi/overlays/development/kustomization.yaml`
      Install RHTAP `./hack/bootstrap-cluster.sh preview --keycloak --toolchain`
      Verify after 5 min there is a data on Grafana dashboards.
```
quay.io/redhat-appstudio/pull-request-builds:spi-controller-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
